src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -373,6 +375,7 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
                 case MessageID.IDS_FeatureCheckedUserDefinedOperators: // semantic check for declarations, parsing check for doc comments
                 case MessageID.IDS_FeatureUTF8StringLiterals: // semantic check
                 case MessageID.IDS_FeatureUnsignedRightShift: // semantic check for declarations and consumption, parsing check for doc comments
+                case MessageID.IDS_FeatureFileTypes: // syntax check
```

Comment:
can you make this a semantic check instead?

Reply:
I suppose the following is more fitting since we moved the LangVersion check to binding, and it checks the DeclarationModifiers rather than SyntaxKind.

```suggestion
                case MessageID.IDS_FeatureFileTypes: // semantic check
```